src/apps/profiles/helpers.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+
+
+def send_mail(context=None, from_email=None, html_file=None, text_file=None, subject=None, to_email=None):
```

Comment:
is this function repeated?

Reply:
You made a `helpers.py` file in the forums PR to define `send_mail`. So I just copied that file to the `profiles` app.